polly/test/Support/dumpmodule.ll

```diff
@@ -68,8 +68,7 @@ return:
 
 
 ; EARLY-LABEL: @callee(
-; AFTEREARLY:  polly.split_new_and_old:
-; EARLY:         store double 4.200000e+01, ptr %arrayidx
+; AFTEREARLY:  polly.stmt.body:
```

Comment:
Can you explain what actually changed in this test? It's hard to understand from this diff.